storage/index.md

```diff
@@ -100,6 +101,10 @@ mounts is to think about where the data lives on the Docker host.
   information. For instance, internally, swarm services use `tmpfs` mounts to
   mount [secrets](/engine/swarm/secrets.md) into a service's containers.
 
+- **[names pipes](https://docs.microsoft.com/en-us/windows/desktop/ipc/named-pipes)**: A `npipe`
+ mount can be used for communication between the Docker host and a container. Common use case is
+ run 3rd party tool inside of container and connect to Docker Engine API using named pipe.
```

Comment:
Some small nits;

- "run _a_ third-party tool .."
- "connect to _the_ Docker..."
- "using _a_ named pipe"

I see some more small issues; let me post the whole paragraph with those fixed so that you can copy/paste;

```Markdown
 **[named pipes](https://docs.microsoft.com/en-us/windows/desktop/ipc/named-pipes)**: An `npipe`
mount can be used for communication between the Docker host and a container. Common use case is
to run a third-party tool inside of a container and connect to the Docker Engine API using a named pipe.
```